operators/src/upgrade/k8s/crd/v0.rs

```diff
@@ -80,15 +80,15 @@ impl UpgradeActionSpec {
 /// 'UpgradePhase' defines the status of each components.
 #[derive(Deserialize, Serialize, Clone, Debug, Eq, PartialEq, JsonSchema)]
 pub enum UpgradePhase {
-    /// Components in Waiting phase.
+    // Components in Waiting phase.
```

Comment:
Why did you remove the doc comment?

Reply:
Because, the kube-rs library uses the doc comment as input for the 'description' field in the CRD openapi spec json/yaml. That description is not openapi3 compliant, and leads to an invalid openapi3 spec.

Reply:
oh I see.. what would it take to make it compliant, have description would actually be good to have?